bounce/db/__init__.py

```diff
@@ -18,7 +18,6 @@ def create_engine(driver, user, password, host, port, db_name):
     return sqlalchemy.create_engine(
         f'{driver}://{user}:{password}@{host}:{port}/{db_name}', echo=True)
 
-
```

Comment:
I don't think this space is supposed to have been removed. You're supposed to have two newlines between functions at the module level.